File: hyrc_base/src/main/java/com/lrs/hyrc_base/view/LoadBaseDialog.java
package com.lrs.hyrc_base.view;

import android.app.Dialog;
import android.content.Context;
import android.os.Bundle;
import android.view.Window;

import com.lrs.hyrc_base.R;

import androidx.annotation.NonNull;


public class LoadBaseDialog extends Dialog {
    public LoadBaseDialog(@NonNull Context context) {
        super(context, R.style.TransparentDialog);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.load_indicator_view);
    }

    public void setMsg(String text) {

    }

}

File: app/src/main/java/com/lrs/lrsmeeting/view/LoadBaseDialog.kt
package com.lrs.lrsmeeting.view

import android.app.Dialog
import android.content.Context
import android.os.Bundle
import android.view.Window
import com.lrs.lrsmeeting.R
import kotlinx.android.synthetic.main.load_indicator_view.*

/**
 * @description 作用:
 * @date: 2020/4/7
 * @author: 卢融霜
 */
class LoadBaseDialog(context: Context) : Dialog(context, R.style.TransparentDialog) {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.load_indicator_view)
    }


    fun setMsg(text: String?) {
        tvLoadName.text = text;
    }
}